### ISSUE TITLE
install docs flawed

### ISSUE THREAD

**Issue by Maxhodges** [Semantic-Org/Semantic-UI#4090]
install didn't work as described because package.json needs to exist. Docs should instruct people to run `npm init` first.

**Comment by Maxhodges** [comment]
also this repo says 

recommended install
`npm install semantic-ui `

yes the semantic-ui webpage says 
`npm install semantic-ui --save  `

shouldn't these two pages agree?

**Comment by IonutBajescu** [comment]
For your first issue, as I recall, `npm init` is not required. What version of npm do you have? 

For the second one, I agree that the save flag should have some kind of consistency, although it's not a so important issue. 
I'll try see which one is more appropriate and release a pull request, with some luck we'll get it sorted out soon.

**Comment by Maxhodges** [comment]
When I followed the instructions as written, the result was errors due to lack of a package.json file. Fortunately a colleague of mind understood that npm init would create that file so he told me to run npm init before doing the semantic install and that succeeded. 
I can check my version of npm when I'm back at the office. 
I just hope we can make getting started with semantic easier for others, since I'd had a very hard time with it (various other issues too, not just this install issue).

Cheers,
Max




On Sun, Jun 12, 2016 at 6:06 AM +0900, "Ionut Bajescu" <notifications@github.com> wrote:












For your first issue, as I recall, npm init is not required. What version of npm do you have? 



For the second one, I agree that the save flag should have some kind of consistency, although it's not a so important issue. 

I'll try see which one is more appropriate and release a pull request, with some luck we'll get it sorted out soon.

**Comment by fuzenco** [comment]
I ran into the same issue as @Maxhodges 

Running `npm init` did it but since I'm new to Node / nPM, I didn't understand all the questions that were being asked. So I did a little research and run `npm install -f` to install with defaults and not be prompted for answers to the questions.

A little more clarity in the install would definitely help out newbies.

Looking forward to working with Semantic. Thanks for your efforts.
